src/mailstoreobserver.cpp

```diff
@@ -390,7 +395,7 @@ void MailStoreObserver::updateMessages(const QMailMessageIdList &ids)
 
 void MailStoreObserver::transmitCompleted(const QMailAccountId &accountId)
 {
-    QList<QObject*> publishedNotifications = _notification->notifications();
+    QList<QObject*> publishedNotifications = Notification::notifications();
```

Comment:
Yes, the default is to only return notifications whose 'appName' property matches the current app.